pyannote/audio/tasks/segmentation/speaker_diarization.py

```diff
@@ -567,7 +569,7 @@ def validation_step(self, batch, batch_idx: int):
             on_epoch=True,
             prog_bar=True,
             logger=True,
-        )
+       )
```

Comment:
This change does not seem right. This probably breaks indentation.

One way to avoid this is to setup precommit hook like explained [here](https://github.com/pyannote/pyannote-audio?tab=readme-ov-file#development)